portal/templates/user/change-password.njk

```diff
@@ -35,13 +35,18 @@
 
         {% include "_partials/password-create.njk" %}
 
+        {% if user._id %}
+          {% set userId = user._id %}
+        {% else %}
+          {% set userId = '' %}
```

Comment:
I assume we should not be able to change the password if the userId is set to `''`?

Reply:
This is just for the cancel button, since we are using this screen for both the scenarios:
1. User actioned reset password (when `_id` will have the user profile ID)
2. Admin creates an account, user sets the password (when `_id` will be empty)

Above is done simply to ensure `Cancel` button does not have `undefined` in the link.